src/lwc/rd2PauseForm/rd2PauseForm.html

```diff
@@ -0,0 +1,105 @@
+<template>
```

Comment:
@rjanjanam , can you please verify elements on the page can be used for automation tests? Thanks.

Reply:
Can see the data-qa-locators and access them. Looks good. Thanks @RadSFDeveloper

Reply:
I do find the use of _both_ a `data-id` and `data-qa-locator` that are so very similar, yet slightly different, to be confusing. Is there value in having two different `data` attributes used for similar purposes, but in different areas? For example, the JS controller can just as easily query for elements by `data-qa-locator` as it can by `data-id`. If there is a desire to keep these as two completely different `data` attributes, it's probably worth spelling this out in a shared document somewhere to define how each are should be used by developers.